roles/search/templates/solrcore.properties.j2

```diff
@@ -1,8 +1,13 @@
 #
 # solrcore.properties - used in solrconfig.xml
 #
-
+{% if search_topology == 'replication' and inventory_hostname != search_master %}
+enable.alfresco.tracking=false
+alfresco.cron=* * * * * ? 1900
+alfresco.nodestate.tracker.cron=* * * * * ? 1900
```

Comment:
not sure what is means

Reply:
It's an ugly way to disable Alfresco tracking. Setting `enable.alfresco.tracking=false` SHOULD do the trick but I suspect it does really work since we introduced support for dynamic registration of shards... Hence this